libs/island-ui/core/src/lib/Input/Input.tsx

```diff
@@ -208,59 +267,14 @@ export const Input = forwardRef(
                 )}
               </label>
             )}
-            <InputComponent
-              className={cn(
-                styles.input,
-                resolveResponsiveProp(
-                  backgroundColor,
-                  styles.inputBackgroundXs,
-                  styles.inputBackgroundSm,
-                  styles.inputBackgroundMd,
-                  styles.inputBackgroundLg,
-                  styles.inputBackgroundXl,
-                ),
-                styles.inputSize[size],
-                {
-                  [styles.rightAlign]: rightAlign,
-                  [styles.textarea]: textarea,
-                },
-              )}
-              id={id}
-              disabled={disabled}
-              name={name}
-              ref={mergedRefs}
-              placeholder={placeholder}
-              value={value}
-              maxLength={maxLength}
-              defaultValue={defaultValue}
-              onFocus={(e) => {
-                setHasFocus(true)
-                if (onFocus) {
-                  onFocus(e)
-                }
-              }}
-              onClick={(e) => {
-                if (onClick) {
-                  onClick(e)
-                }
-              }}
-              onKeyDown={(e) => {
-                if (onKeyDown) {
-                  onKeyDown(e)
-                }
-              }}
-              onBlur={(e) => {
-                setHasFocus(false)
-                if (onBlur) {
-                  onBlur(e)
-                }
-              }}
-              readOnly={readOnly}
-              type={type}
-              {...(ariaError as AriaError)}
-              {...inputProps}
-              {...(required && { 'aria-required': true })}
-            />
+            {prefix ? (
+              <Box display="flex" alignItems="center">
+                <span className={styles.prefix}>{prefix}</span>
+                {input}
+              </Box>
+            ) : (
+              input
+            )}
```

Comment:
The parent element already has display flex and alignItems center. Do we need this extra Box?

Reply:
Yes this is necessary the parent element is, ` <Box flexGrow={1}>` but the grandparent is what you where referring to